lightning/src/util/config.rs

```diff
@@ -265,30 +291,6 @@ pub struct ChannelConfig {
 	///
 	/// [`MIN_CLTV_EXPIRY_DELTA`]: crate::ln::channelmanager::MIN_CLTV_EXPIRY_DELTA
 	pub cltv_expiry_delta: u16,
-	/// Set to announce the channel publicly and notify all nodes that they can route via this
```

Comment:
nit: If you split up the commits, you may want to move the removal of the moved parts to the 'move' commits. (Nevermind, if you end up squashing it all).